v2/pl/pl_holidays_test.go

```diff
@@ -101,6 +101,16 @@ func TestHolidays(t *testing.T) {
 		{NationalIndependenceDay, 2021, d(2021, 11, 11), d(2021, 11, 11)},
 		{NationalIndependenceDay, 2022, d(2022, 11, 11), d(2022, 11, 11)},
 
+		{ChristmasEve, 2024, d(2025, 12, 24), d(2025, 12, 24)},
```

Comment:
This test should return a zero time, right?

Reply:
You're right, I fixed it in the last commit.